.gitignore

```diff
@@ -1,4 +1,5 @@
+.eslintrc.js
+*.ts
```

Comment:
Is this intentional? So every new source file must be explicitly added to commit it? I don't think this is a good idea. It is easy to forget to add a file and then an incomplete change is committed and will cause trouble or at least some confusion.

A file which was accidentally committed can be fixed by anyone everywhere without much effort. I missing file can only be fixed by the author and only on the machine where the file was written. And an accidentally added file is easily spotted when reviewing the changes locally before committing them and are also easily spotted when reviewing a PR. A missing file may be hard to notice on all sides. Even a CI build may not detect it when the file was a unit test file for example which is then simply not executed.

Reply:
I was hoping to exclude only top level .ts files. I will correct this.